superset/assets/javascripts/explore/components/ControlPanelsContainer.jsx

```diff
@@ -65,14 +64,15 @@ class ControlPanelsContainer extends React.Component {
                 <ControlRow
                   key={`controlsetrow-${i}`}
                   controls={controlSets.map(controlName => (
-                    <Control
-                      name={controlName}
-                      key={`control-${controlName}`}
-                      value={this.props.form_data[controlName]}
-                      validationErrors={this.props.controls[controlName].validationErrors}
-                      actions={this.props.actions}
-                      {...this.getControlData(controlName)}
-                    />
+                    controlName &&
```

Comment:
I introduced support for `null` in `visTypes.jsx` as a way to use half of the grid